lib/wickedbase.pm

```diff
@@ -20,6 +20,8 @@ use testapi qw(is_serial_terminal :DEFAULT);
 use serial_terminal;
 use Carp;
 use Mojo::File 'path';
+use serial_terminal 'upload_file';
```

Comment:
You are already exporting it by default, and in line 20 it is already imported :D

Reply:
Oh, thx I overseen it :)
What do you mean with `exporting it by default`. I only see `@EXPORT` in ` serial_terminal.pm`.

Reply:
Any symbol inside `@EXPORT` will be imported when `use Foo;` is used :), `@EXPORT_OK` on the other hand behaves a little bit different, as it will allow the module to export those symbols, but on demand (`use Mojo::File qw(path)` for example...)

Reply:
@foursixnine thx for pointing me at this. I twisted it.
What would you recommend, using the `:DEFAULT` export as long as it exists?